index.js

```diff
@@ -11,24 +12,33 @@ const Wilson = module.exports = function (config) {
     return new Wilson(config)
   }
 
-  this.logger = Pino()
+  this.id = HyperId().uuid
+  this.log = Pino()
```

Comment:
Happy with this but a PR will need to be made on the transport as the logger is passed down